tests/foreman/api/test_ping.py

```diff
@@ -29,6 +29,7 @@ def test_positive_ping(target_sat):
 
     :expectedresults: Overall and individual services status should be 'ok'.
     """
+    logger.info("xxx inside test body")
```

Comment:
same comment as before
```suggestion
    logger.info("inside test body")
```